src/utils/viewer/viewer.cpp

```diff
@@ -536,11 +536,11 @@ Viewer::Viewer(const Arguments& arguments)
 
   // Set up camera
   activeSceneGraph_ = &simulator_->getActiveSceneGraph();
-  renderCamera_ = &activeSceneGraph_->getDefaultRenderCamera();
-  renderCamera_->setAspectRatioPolicy(
-      Mn::SceneGraph::AspectRatioPolicy::Extend);
   defaultAgent_ = simulator_->getAgent(defaultAgentId_);
   agentBodyNode_ = &defaultAgent_->node();
+  renderCamera_ = getAgentCamera().getRenderCamera();
+  renderCamera_->setAspectRatioPolicy(
```

Comment:
now since the renderCamera is embedded in the sensor. 
 please
 `setAspectRatioPolicy(
      Mn::SceneGraph::AspectRatioPolicy::Extend)`
      in the CameraSensor constructor, after you create the renderCamera.